metadata-ingestion/src/datahub/ingestion/source/schema_inference/json.py

```diff
@@ -1,6 +1,7 @@
 from typing import IO, Dict, List, Type, Union
 
 import ujson
+import newlinejson as nlj
```

Comment:
I'd have a slight preference towrads [jsonlines](https://github.com/wbolster/jsonlines) package - since it seems a bit better maintained